### ISSUE TITLE
[A11y] The Tooltip content can't be read by JAWS

### ISSUE THREAD

**Issue by icemanlily** [IBM/carbon-components-react#2063]
===== ENVIRONMENT ===== 
IBM CI 162 Checkpoint Violation: 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html
Build Level: Published - 02/27/2019
Test Tool Type: JAWS

===== DEFECT DESCRIPTION ===== 
The tooltip information cannot be read by screen reader, it only read as "tooltip button menu... ", for both "Select a resource group" and "Tags"

=====STEPS TO REPRODUCE ===== 

Login to IBM Cloud stage env
Navigate to Catalog -> Security and Identity -> Hyper Protect Crypto Service
Use Tab key to navigate to the tooltip icon

![image](https://user-images.githubusercontent.com/6970471/54661144-08c3f580-4b14-11e9-91ee-db914e6e74f8.png)

**Comment by icemanlily** [comment]
Fixed and close this issue.
